pages/components/index.mdx

```diff
@@ -40,7 +40,7 @@ export default () => (
 )
 ```
 
-This will set the `color`, `font-family`, and `line-height` CSS properties to the same ones used in [primer-base](https://github.com/primer/primer/blob/master/modules/primer-base/lib/base.scss#L15).
+This will set the `color`, `font-family`, and `line-height` CSS properties to the same ones used in [primer-base](https://github.com/primer/css/blob/master/src/base/base.scss#L15).
```

Comment:
I wonder if we should update the terminology away from the `primer-*` language we've been using. @shawnbot ?

Reply:
Maybe should say `@primer/css/base` instead?

Reply:
@chhuang Mind committing this suggestion? I can't since I don't have access to your repo! After that I'll merge this :) Thank you again!

Reply:
Sorry I overlooked this comment, I should have ticked the box to allow edits from maintainers.